### DISCUSSION TITLE
Let me know  how to configure LDAP replication in a cloud environment.

### DISCUSSION THREAD

**Discussion by yavis73** [OpenIdentityPlatform/OpenDJ#319] [General]
Do you know how to automatically deploy a Replication configuration when configuring a cloud service using openDJ?

**Comment by maximthomas** [comment]
Hi @yavis73, these links could be heplful:
https://github.com/OpenIdentityPlatform/OpenDJ/wiki/How-To-Run-OpenDJ-in-Kubernetes
https://github.com/OpenIdentityPlatform/OpenDJ/wiki/Administration-Replication
https://github.com/OpenIdentityPlatform/OpenDJ/wiki/How-To#initialize-replication-between-two-servers
There is also the `OPENDJ_REPLICATION_TYPE` environment variable for OpenDJ Docker distribution. 
More details about OpenDJ Docker environment variables are here: https://hub.docker.com/r/openidentityplatform/opendj/. If you want to dive deeper into the details see [opendj-docker/bootstrap/replicate.sh](https://github.com/OpenIdentityPlatform/OpenDJ/blob/master/opendj-packages/opendj-docker/bootstrap/replicate.sh) file
